### PR TITLE
Update code to reflect true requirements

### PR DESCRIPTION
**Pull request by SteveScorfield**

### CODE REVIEW

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java [hunk 1]
```diff
@@ -190,8 +191,19 @@ private static void createIndefinitePause(CancelFieldWorkerJobRequest cancelJobR
   }
 
   public static UpdateFieldWorkerJobRequest newUpdateJob(ActionInstruction actionInstruction) {
+    ActionUpdate actionUpdate = actionInstruction.getActionUpdate();
+
     UpdateFieldWorkerJobRequest updateJobRequest = new UpdateFieldWorkerJobRequest();
     updateJobRequest.setActionType("update");
+    updateJobRequest.setId(UUID.fromString(actionUpdate.getCaseId()));
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java]: should be updateJobRequest.setCaseId(...) Ive update datamap, the change needs to happen in canonical. Im keeping it consistent with Action Request

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java [hunk 2]
```diff
@@ -190,8 +191,19 @@ private static void createIndefinitePause(CancelFieldWorkerJobRequest cancelJobR
...
+    updateJobRequest.setAddressType(actionUpdate.getAddressType());
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java:199]: change to updateJobRequest.setEstablishmentType(...)

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java [hunk 3]
```diff
@@ -190,8 +191,19 @@ private static void createIndefinitePause(CancelFieldWorkerJobRequest cancelJobR
...
+    updateJobRequest.setAddressLevel(actionUpdate.getAddressLevel());
+    updateJobRequest.setUndeliveredAsAddressed(actionUpdate.isUndeliveredAsAddress());
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java]: change to updateJobRequest.setUaa(...)

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java [hunk 4]
```diff
@@ -190,8 +191,19 @@ private static void createIndefinitePause(CancelFieldWorkerJobRequest cancelJobR
...
+    updateJobRequest.setUntil(convertXmlGregorianToOffsetDateTime(actionUpdate.getActionableFrom()));
+    updateJobRequest.setCe1Complete(actionUpdate.isCe1Complete());
+    updateJobRequest.setCeExpectedResponses(actionUpdate.getCeExpectedResponses().intValue());
+    updateJobRequest.setCeActualResponses(actionUpdate.getCeActualResponses().intValue());
+    updateJobRequest.setBlankQreReturned(actionUpdate.isBlankQreReturned());
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java]: updateJobRequest.setBlankFormReturned(...)

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java [hunk 5]
```diff
@@ -190,8 +191,19 @@ private static void createIndefinitePause(CancelFieldWorkerJobRequest cancelJobR
...
   public static UpdateFieldWorkerJobRequest newUpdateJob(ActionInstruction actionInstruction) {
+    ActionUpdate actionUpdate = actionInstruction.getActionUpdate();
+
     UpdateFieldWorkerJobRequest updateJobRequest = new UpdateFieldWorkerJobRequest();
     updateJobRequest.setActionType("update");
+    updateJobRequest.setId(UUID.fromString(actionUpdate.getCaseId()));
+    updateJobRequest.setAddressType(actionUpdate.getAddressType());
+    updateJobRequest.setAddressLevel(actionUpdate.getAddressLevel());
+    updateJobRequest.setUndeliveredAsAddressed(actionUpdate.isUndeliveredAsAddress());
+    updateJobRequest.setUntil(convertXmlGregorianToOffsetDateTime(actionUpdate.getActionableFrom()));
```
Comment (sdiaz1000) [inline src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java]: change to updateJobRequest.setHoldUntil(...)

**Review by sdiaz1000 — DISMISSED** [review]

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java (hunk 1 above)
Comment (SteveScorfield) [reply src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java]: Have update this and have updated canonical.

File: src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java (hunk 2 above)
Comment (sdiaz1000) [reply src/main/java/uk/gov/ons/census/fwmt/rmadapter/canonical/CanonicalJobHelper.java:199]: In the create this field is estabType but in here it address type, can you change it to estabType to keep it consistent

**Review by sdiaz1000 — APPROVED** [review]

**Pull request merged**